### PR TITLE
feat: professor rating slider

### LINKED ISSUE #1563: Professor score sliding bar
There is a sliding bar for overall score and for workload score on the course search, but would be great if one existed for prof ratings too

https://coursetable.canny.io/admin/board/feature-requests/p/professor-score-sliding-bar

### PR DESCRIPTION
**Pull request by bearsyankees**
Fixes #1563 to add a professor slider score

### CODE REVIEW

**Comment by Josh-Cena** [comment]
@bearsyankees BTW, if you use "Fixes #1563" in your PR description it will auto-close the issue

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
image

There's a problem on tablet view (the reset button gets pushed to the next line). Maybe we should always put professor under advanced?

**Comment by bearsyankees** [comment]
Yeah I think that's a good idea will do later may be offline a bit today and tom tho

**Comment by Josh-Cena** [comment]
No hurries :)

**Comment by bearsyankees** [comment]
> image
> 
> There's a problem on tablet view (the reset button gets pushed to the next line). Maybe we should always put professor under advanced?

always meaning even on desktop view?

**Comment by Josh-Cena** [comment]
Yeah, but only hiding it on tablet is fine for me too

File: frontend/src/components/Search/MobileSearchForm.tsx [hunk 1]
```diff
@@ -207,6 +208,33 @@ export default function MobileSearchForm({
...
+                    <Handle {...e} key={e.className}>
+                      <div className={clsx('shadow', styles.workload_tooltip)}>
+                        {value}
+                      </div>
+                    </Handle>
+                  )}
+                  className={styles.slider}
+                />
+              </Container>
+              <div className={clsx('text-center', styles.filter_title)}>
```
Comment (Josh-Cena) [inline frontend/src/components/Search/MobileSearchForm.tsx:234]:
```suggestion
              <div className={clsx('text-center', styles.filterTitle)}>
```

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**